tests/unit_tests/test_image_bounder.py

```diff
@@ -0,0 +1,67 @@
+from SyntheticDataset.image_operations import *
+import unittest
+from PIL import Image
+
+class ImageBounderTestCase(unittest.TestCase):
+
+
+    def test_getBoundsOfColor(self):
+        self.test_img1 = Image.open("tests/images/test_image2.png")
+        self.test_image1 = self.test_img1.load()
```

Comment:
If you are going to assign the loaded image to the same variable that contained the original image, condense the two calls into one line.

Reply:
i do need both img and image here

Reply:
Ah yes you are correct I apologize...For some reason my brain thought those two images were both the same name...